### DISCUSSION TITLE
Streaming with nova-2-medical and language="hi" gives 400 BAD REQUEST

### DISCUSSION THREAD

**Discussion by aagmangour** [deepgram/community#579] [General help] [answered]
Hello,

I have been trying my hands out on deepgram using the python sdk. I intent to use the Streaming option, while using the model= nova-2-medical with and without "en-US" things works fine. Here is the API request :
**POST /v1/listen?channels=1&encoding=linear16&interim_results=true&model=nova-2-medical&punctuate=true&sample_rate=16000&smart_format=true**
Request ID: ed0c5f95-305e-4094-b9c2-8496193233d2
Status
101 OK

But If I use nova-2-medical with the language selection as "hi", it gives me 400 Bad Request :

**POST /v1/listen?channels=1&encoding=linear16&interim_results=true&language=hi&model=nova-2-medical&punctuate=true&sample_rate=16000&smart_format=true**
Request ID :1f75ee9b-dc03-49c6-b0ce-8907ace9e982
Status
400
Performed at
02/02/2024 06:26:19
Error: "Bad Request"

However, with the nova-2-general, I was able to use language="hi"

**POST /v1/listen?channels=1&encoding=linear16&interim_results=true&language=hi&model=nova-2-general&punctuate=true&sample_rate=16000&smart_format=true**
Request ID
19f8e20a-29c9-4762-97cc-04473e56afce
Status
101 OK

Is there any limitation on languages while using nova-medical.

Thanks

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by aagmangour** [comment, marked as answer]
Got the answer from another  post:  https://github.com/orgs/deepgram/discussions/493#discussioncomment-7865891

`We offer the medical model for US English (language=en-US) as well as global English (language=en). That should generally improve accuracy of your medical transcription`

Follow up question: If I plan to use "keyword" is there any limit on number of keywords that can be passed for a single request

**Reply by jkroll-deepgram** [reply]
Yes, there is a limit of 200 keywords.

Glad that you were able to find the answer to your question!
